src/porepy/applications/test_utils/common_xpfa_tests.py

```diff
@@ -0,0 +1,740 @@
+"""Utility functions used here in test_mpfa.py and test_tpfa.py.
+Some simple for getting grid, permeability, bc object etc.
+
+Then more specific functions related to specific tests defined both here and for mpfa.
+
+"""
+import numpy as np
+import scipy.sparse.linalg as spla
+
+import porepy as pp
+
+
+def _setup_cart_2d(nx, dir_faces=None):
+    g = pp.CartGrid(nx)
+    g.compute_geometry()
+    kxx = np.ones(g.num_cells)
+    perm = pp.SecondOrderTensor(kxx)
+    if dir_faces is None:
+        dir_faces = g.tags["domain_boundary_faces"].nonzero()[0]
```

Comment:
```suggestion
        # If no Dirichlet faces are specified, set Dirichlet conditions on all faces.
        dir_faces = g.tags["domain_boundary_faces"].nonzero()[0]
```
Or make a similar comment if my attempt is not correct.